.circleci/config.yml

```diff
@@ -131,6 +131,8 @@ jobs:
             pip install -U pip
             pip install -e .
```

Comment:
This works, suggestion:

Use `pip install -e .[binder]` and add a "binder" section to the extras in `setup.py` much like the current `tests` section. Example in line 18 of this file where tests requirements are installed. you can remove all 3 (L 133, 134, 145) pip installs here after moving the requirements to the extra section.